spec/models/dex_connector_ldap_spec.rb

```diff
@@ -18,4 +18,362 @@
         .to include("BEGIN CERTIFICATE")
     end
   end
+
+  describe "#name_validations" do
+    let(:dex_connector_ldap) { create(:dex_connector_ldap) }
+
+    it "allows the name to be present" do
+      dex_connector_ldap.name = "openldap"
```

Comment:
There are a few rspec built-in matchers [0] that reduces the noise for this kinda of validation. For example:

```ruby
it { is_expected.to validate_presence_of(:name) }
```

[0] https://www.rubydoc.info/github/thoughtbot/shoulda-matchers/master/Shoulda/Matchers/ActiveModel

Reply:
Cool I'll take a look. Looks like I have a couple of tests to fix as well. I'll get it taken care of asap.